src/main/java/duke/exception/EmptyTaskException.java

```diff
@@ -0,0 +1,4 @@
+package duke.exception;
+
+public class EmptyTaskException extends Throwable {
```

Comment:
From the Javadoc https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html, "The Throwable class is the superclass of all errors and exceptions in the Java language". In this case, since EmptyTaskException should be an Exception and not an Error, it is recommended to extend from the Exception class rather than Throwable, which includes both Exception and Error.